DigitalLearningSolutions.Web/Views/NotificationPreferences/_NotificationPreferenceExpander.cshtml

```diff
@@ -8,7 +9,7 @@
 <details class="nhsuk-details nhsuk-expander">
   <summary class="nhsuk-details__summary">
     <span class="nhsuk-details__summary-text">
-      My @(Model.UserType == UserTypes.Admin ? "admin" : "delegate") account notifications
+      My @(((UserType)Model.UserType).Equals(UserType.AdminUser) ? "admin" : "delegate") account notifications

```

Comment:
I'm surprised the type doesn't get inferred automatically 👁️

Reply:
Ah that's fair! I think swapping them around would be nicer, just to avoid the extra brackets, which aren't very easy to read. However, if you find it more difficult to read that way, then maybe the answer is to just pull this whole line into a variable 🤔

Reply:
Ah, I tell a lie, you still need the explicit cast - `Enumeration.Equals()` takes an `object?`.

Reply:
In that case, let's leave it! 👍